static/js/tinderAMatch.js

```diff
@@ -0,0 +1,84 @@
+let readingList = [];
```

Comment:
Since we're not using any build tools, I would avoid ES6 in front-end JS to improve browser compatibility.